src/eachobs.jl

```diff
@@ -1,21 +1,29 @@
 """
     eachobs(data, buffer=false, batchsize=-1, partial=true)
```

Comment:
```suggestion
    eachobs(data; [buffer, batchsize, partial, parallel, shuffle])
```
and defaults should be mentioned in the list of arguments